src/corelibs/U2View/src/ov_msa/ColorSchemaDialogController.cpp

```diff
@@ -382,17 +383,35 @@ void ColorSchemaSettingsPageWidget::sl_schemaChanged(int index) {
     }
 }
 
+bool ColorSchemaSettingsPageWidget::sl_validateSchemesDir() {
+    QString dirPath = colorsDirEdit->text();
```

Comment:
Use GUrlUtils::prepareDirLocation()

Reply:
Ensure that prepareDirLocation() is called only once at the end and not for every path entered by user